src/main/java/com/faforever/api/data/domain/Avatar.java

```diff
@@ -25,12 +25,13 @@
   private List<AvatarAssignment> assignments;
 
   @Column(name = "url")
-  @NotNull
+//  @NotNull
```

Comment:
Why commented out? Add back, remove, or explain

Reply:
ah sry this is left over from some experiments